resources/lang/id/response.php

```diff
@@ -13,5 +13,6 @@
     */
 
     'sync_failed' => 'Data Gagal Dikirim ke Aplikasi SIM Lab',
-    'sync_success' => 'Data Berhasil Terkirim'
+    'sync_success' => 'Data Berhasil Terkirim',
+    'export_failed' => 'Ekspor Gagal, Silahkan lengkapi Kode Sample untuk dapat melakukan Ekspor.'
```

Comment:
Minor kang. untuk localization string-nya, apakah memang sengaja diabuat title case?

Kalau iya, sepertinya semua kata yg sesuai harus dibuat title case

```suggestion
    'export_failed' => 'Ekspor Gagal, Silahkan Lengkapi Kode Sample untuk Dapat Melakukan Ekspor.'
```

Reply:
sebetulnya kalo liat kalimat2 yang existing nggak `tittle case `siii kang :smile:
mungkin biar seragam ku coba ubah dulu kang @aldidoanta  :+1:

Reply:
but waittt kalo yg versi en itu non-title case, versi in itu `title-case` :smile: 

any suggestion kang bagusnya untuk response message ini `tittle-case `atau `non-tittle-case` yaa? biar sekalian dirapihkan juga yg lainnya :+1: :grimacing:

Reply:
Kalau menurut aku pribadi, lebih baik tidak perlu pakai title case. Hanya beberapa kata terkait fitur aja yang dibuat title case, misalnya `Kode Sample` atau `Aplikasi SIM Lab`